internal/packages/conditions.go

```diff
@@ -73,7 +73,7 @@ func parsePackageRequirements(keyValuePairs []string) (*packageRequirements, err
 			}
 			pr.kibana.version = &withoutPrerelease
 		default:
-			return nil, fmt.Errorf("unknown package requirement: %s", s[0])
+			return nil, fmt.Errorf("unknown package requirement: %s", value)
```

Comment:
According to the previous error message, it should be set here `key` , no? 

```suggestion
			return nil, fmt.Errorf("unknown package requirement: %s", key)
```